src/applications/mhv/medications/components/shared/FillRefillButton.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { fillPrescription } from '../../actions/prescriptions';
+
+const FillRefillButton = rx => {
+  const dispatch = useDispatch();
+  const [isAlertVisible, setAlertVisible] = useState(false);
+
+  const {
+    cmopDivisionPhone,
+    dispensedDate,
+    isRefillable,
+    prescriptionId,
+    refillStatus,
+  } = rx;
+
+  // TODO: This is what the logic used to be before. Needs to be updated
+  if (refillStatus === 'expired' || refillStatus === 'refillinprocess') {
+    return null;
+  }
+
+  if (isRefillable || (dispensedDate && !isRefillable)) {
+    return (
+      <div className="no-print">
+        {/* TODO: Confirmation Message goes here */}
+        <div hidden={!isAlertVisible}>
```

Comment:
What do you think of doing something like this instead:

```js
{isAlertVisible &&
  <va-alert status="error">
    ...
```

This way it doesn't have to be rendered and hidden and it simplifies the code a little bit 🙂

Reply:
I agree. This will come with a separate PR when the Fill/Refill button visibility logic is updated. Thank you.